### PR TITLE
Remove postfix New (only) from sampler names

### LINKED ISSUE #478: Remove postfix New from sampler names

### PR DESCRIPTION
**Pull request by chaozg**
Fixed #478 

TLDR:
`cuqi.experimental.NUTSNew` -> `cuqi.experimental.NUTS`

### CODE REVIEW

File: tests/zexperimental/test_mcmc.py [hunk 1]
```diff
@@ -92,7 +92,7 @@ def assert_true_if_warmup_is_equivalent(
         samples_new = \
             sampler_new.get_samples().samples[...,Nb+new_idx[0]:new_idx[1]]
 
-    assert np.allclose(samples_old, samples_new), f"Old: {samples_old[0]}\nNew: {samples_new[0]}"
```
Comment (nabriis) [inline tests/zexperimental/test_mcmc.py:95]: This seems to be an error

File: tests/zexperimental/test_mcmc.py [hunk 2]
```diff
@@ -34,11 +34,11 @@ def assert_true_if_sampling_is_equivalent(
         sampler_new.sample(Ns).get_samples().samples[..., new_idx[0]:new_idx[1]]
     )
 
-    assert np.allclose(samples_old, samples_new, atol=atol), f"Old: {samples_old}\nNew: {samples_new}"
```
Comment (nabriis) [inline tests/zexperimental/test_mcmc.py:37]: Also here an error.

**Review by nabriis — APPROVED** [review]
Hi @chaozg. LGTM. I found 2 minor mistakes.

File: tests/zexperimental/test_mcmc.py (hunk 1 above)
Comment (chaozg) [reply tests/zexperimental/test_mcmc.py]: Corrected

File: tests/zexperimental/test_mcmc.py (hunk 2 above)
Comment (chaozg) [reply tests/zexperimental/test_mcmc.py]: Corrected

**Review by amal-ghamdi — CHANGES_REQUESTED** [review]
Thank you @chaozg, this is great! 

Just one comment, in the beginning of demo37 there are 2 occurrences in the text for `HybridGibbsNew`. It will be great if you can update that.

```
## HybridGibbsNew
One feature we get from the new sampler design is more flexibility in our Gibbs sampling.
Now we can for example run with sampler instances, which allows us to set parameters and also use all samplers in Gibbs sampling.

We call this new sampler HybridGibbsNew to distinguish it from classical Gibbs sampling.
```

**Comment by chaozg** [comment]
Thank you, Amal, for noticing that. I just pushed an update.

**Review by amal-ghamdi — APPROVED** [review]
Nice @chaozg, you also spotted the third occurrence 👍

**Pull request merged**